apps/heureka/src/components/common/SeverityCount.tsx

```diff
@@ -13,19 +13,22 @@ import {
   BadgeVariantType,
 } from "@cloudoperators/juno-ui-components"
 
-type SeverityCountProps = { count: number; icon: KnownIcons; variant: BadgeVariantType; tooltipContent: string }
+type SeverityCountType = {
+  icon?: KnownIcons
+  count: number
+  variant: BadgeVariantType
+  tooltipContent: string
+  className?: string
+}
 
-export const SeverityCount = ({ count, icon, variant, tooltipContent }: SeverityCountProps) => (
-  <>
-    {count > 0 ? (
-      <Tooltip triggerEvent="hover">
-        <TooltipTrigger>
-          <Badge icon={icon} text={count > 0 ? count.toString() : "-"} variant={count > 0 ? variant : "default"} />
-        </TooltipTrigger>
-        <TooltipContent>{tooltipContent}</TooltipContent>
-      </Tooltip>
-    ) : (
-      <span>&mdash;</span>
-    )}
-  </>
-)
+export const SeverityCount = ({ icon, count, variant, tooltipContent, className }: SeverityCountType) =>
+  count > 0 ? (
+    <Tooltip triggerEvent="hover">
+      <TooltipTrigger>
+        <Badge icon={icon} text={count.toString()} variant={variant} className={className} />
+      </TooltipTrigger>
+      <TooltipContent>{tooltipContent}</TooltipContent>
+    </Tooltip>
+  ) : (
+    <span>&mdash;</span>
+  )
```

Comment:
I'd suggest that you add two test cases one with count and the other one with no count.